### PR TITLE
Fix unquoted path parameters in Microsoft.NET.Sdk.IL.targets with Windows

### PR DESCRIPTION
**Pull request by jasonkuo41**
Fixes #13740

### CODE REVIEW

File: src/coreclr/src/.nuget/Microsoft.NET.Sdk.IL/targets/Microsoft.NET.Sdk.IL.targets [hunk 1]
```diff
@@ -125,17 +125,23 @@ Copyright (c) .NET Foundation. All rights reserved.
 
       <_KeyFileArgument Condition="'$(KeyOriginatorFile)' != ''">-KEY="$(KeyOriginatorFile)"</_KeyFileArgument>
 
+      <_IlasmCommand>$(_IlasmDir)ilasm</_IlasmCommand>
+      <_IlasmCommand Condition="'$(OS)'=='Windows_NT'">"$(_IlasmCommand)"</_IlasmCommand>
+
+      <_IlasmResourceFilePath>$(IlasmResourceFile)</_IlasmResourceFilePath>
+      <_IlasmResourceFilePath Condition="'$(OS)'=='Windows_NT'">"$(_IlasmResourceFilePath)"</_IlasmResourceFilePath>
```
Comment (am11) [inline src/coreclr/src/.nuget/Microsoft.NET.Sdk.IL/targets/Microsoft.NET.Sdk.IL.targets:132]: Double quotes for path with spaces is typically also required on Unix.  Maybe just add quotes directly to line 135 and 138 in the original revision.

Comment (jasonkuo41) [reply src/coreclr/src/.nuget/Microsoft.NET.Sdk.IL/targets/Microsoft.NET.Sdk.IL.targets:132]: #13766 reported that adding quotes directly may break Unix build, thus I made this pull request targeting towards Windows first.

**Comment by ViktorHofer** [comment]
// Auto-generated message

https://github.com/dotnet/runtime/commit/69e114c1abf91241a0eeecf1ecceab4711b8aa62 which was merged 12/7 removed the intermediate src/coreclr/**src**/ folder. This PR needs to be updated as it touches files in that directory which causes conflicts.

To update your commits you can use this bash script: https://gist.github.com/ViktorHofer/6d24f62abdcddb518b4966ead5ef3783. Feel free to use the comment section of the gist to improve the script for others.

**Comment by danmosemsft** [comment]
I'm going to close this PR as it is old now and has conflicts. Thank you for offeringit.

**Pull request closed without merging**